ui/test/unit/registration/controllers/patientCommonController.spec.js

```diff
@@ -5,7 +5,7 @@ describe('PatientCommonController', function () {
     var $aController, $httpBackend, scope, appService, rootScope, patientAttributeService;
     var spinner = jasmine.createSpyObj('spinner', ['forPromise']);
 
-    beforeEach(module('bahmni.registration'));
+    beforeEach(module('bahmni.registration', 'ngDialog'));
```

Comment:
Can you work on a unit test that checks that when the Home button is clicked the confirmation popup is prompted?

Reply:
Yes